frontend/database/145_change_quality_nominations_status.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE `QualityNominations`
+CHANGE COLUMN `status` `status` ENUM('open','approved','denied','warning') NOT NULL DEFAULT 'open' COMMENT 'El estado de la nominación' ;
```

Comment:
```suggestion
CHANGE COLUMN `status` `status` ENUM('open','resolved','banned','warning') NOT NULL DEFAULT 'open' COMMENT 'El estado de la nominación' ;
```

como ahora la nominación puede tener dos resultados de "aceptación" (mover a banneado o advertencia) y uno de resolución (mover a público/privado), creo que es momento de reorganizar esto. también hay que hacer un script de migración para cambiar los valores del enum que ya están guardados en la base de datos.

posiblemente haya que hacer eso en tres pasos:

- agregar `'resolved', 'banned', 'warning'` al enum
- migrar los valores que ya existan
- quitar `'approved', 'denied'`.

igual para la otra tabla.

Reply:
@lhchavez puedo agregar todos los cambios en el mismo archivo .sql?

Reply:
listo